tests/Feature/BestReplyTest.php

```diff
@@ -40,6 +40,24 @@ public function testOnlyThreadCreatorMayMarkReplyAsBest()
 
         $this->assertFalse($replies[1]->fresh()->isBest());
     }
+    
+    /** @test */
+    public function testOnlyThreadCreatorMayUnmarkReplyAsBest()
+    {
+        $this->withExceptionHandling();
+        $this->signIn();
+
+        $thread = create(Thread::class, ['user_id' => Auth::id()]);
+        $replies = create(Reply::class, ['thread_id' => $thread->id], 2);
+
+        $this->signIn();
+
+        $this->deleteJson(route('replies.unmark_best', $replies[1]))
+            ->assertStatus(403);
+
+        $this->assertFalse($replies[1]->fresh()->isBest());
```

Comment:
This should be changed to an assertTrue and the reply should be marked as best before the HTTP DELETE request. In this case, we are not actually checking that the reply's "best status" was left unchanged.